lang/src/parser.rs

```diff
@@ -59,28 +59,32 @@ impl ast::Item {
 
 impl Parse for ast::Item {
     fn parse(input: ParseStream<'_>) -> Result<Self> {
-        let attrs = syn::Attribute::parse_outer(input)?;
+        let inner_attrs: ast::ItemEnvMeta = input.parse()?;
```

Comment:
Can't we make use of https://oschwald.github.io/maxminddb-rust/syn/struct.Attribute.html#method.parse_inner ?
`syn::Attribute::parse_inner(input)` would be the perfect fit for us and then we would need to dispatch those attributes into our own. This could also be used later for proper versioning of smart contracts which has already been planned.

Note that if this is too much work for you, then let's leave this to another PR.

Reply:
Just saw your implementation already goes into a decent direction. So no change needed here.